static/openapi/prerendering-spec.json

```diff
@@ -3060,1025 +3384,1091 @@
+            "description": "Origin hostname."
           },
           {
             "schema": { "type": "string" },
             "in": "header",
             "name": "x-photoniq-customerid",
             "required": true,
-            "description": "Customer unique identifier"
+            "description": "Customer unique identifier."
           }
         ],
         "responses": {
           "200": {
-            "description": "Successful response",
+            "description": "Successful response.",
             "content": {
               "application/json": {
                 "schema": {
                   "type": "object",
-                  "description": "Successful response",
+                  "description": "Successful response.",
                   "properties": {
                     "status": {
                       "type": "string",
-                      "description": "Success status"
+                      "description": "Success status."
                     },
                     "message": {
                       "type": "string",
-                      "description": "Success message"
+                      "description": "Success message."
                     },
                     "data": {
-                      "type": "object",
-                      "properties": {
-                        "_key": {
-                          "type": "string",
-                          "description": "Prefetch configuration unique identifier."
-                        }
-                      },
-                      "required": ["_key"]
+                      "type": "array",
+                      "items": {
+                        "type": "object",
+                        "properties": {
+                          "url": {
+                            "type": "string",
+                            "format": "uri",
+                            "description": "Origin URL."
+                          },
+                          "isMobile": {
+                            "type": "boolean",
+                            "description": "Whether the URL is rendered for mobile devices or not."
+                          },
+                          "status": {
+                            "type": "string",
+                            "enum": [
+                              "queued",
+                              "retrying",
+                              "completed",
+                              "failed"
+                            ],
+                            "description": "URL status."
+                          },
+                          "createdAt": {
+                            "type": "number",
+                            "description": "Timestamp of when the prefetch URL was created."
+                          },
+                          "updatedAt": {
+                            "type": ["null", "number"],
+                            "description": "Timestamp of when the prefetch URL was updated."
```

Comment:
Same for _updatedAt_:  Standardize on one way.

Reply:
Same here